BigBaseV2/src/backend/looped/weapons/no_spread.cpp

```diff
@@ -1,51 +1,50 @@
-#include "backend/looped/looped.hpp"
 #include "pointers.hpp"
 #include "CPedFactory.hpp"
+#include "backend/looped/looped.hpp"
```

Comment:
Set these alphabetically, to follow the same coding style as the entire project.

No need to include CPedFactory.hpp, it's already included automatically by common.hpp